src/Fantomas.Core.Tests/TypeAnnotationTests.fs

```diff
@@ -262,23 +262,11 @@ let private asJson (arm: IArmResource) =
        >
 """
 
-let forcedLongDefnConfig =
-    { config with
-        MaxLineLength = 30
-        SpaceBeforeUppercaseInvocation = true
-        SpaceBeforeClassConstructor = true
-        SpaceBeforeMember = true
-        SpaceBeforeColon = true
-        SpaceBeforeSemicolon = true
-        MultilineBracketStyle = Aligned
-        AlignFunctionSignatureToIndentation = true
-        // AlternativeLongMemberDefinitions = true
-        MultiLineLambdaClosingNewline = true
-        NewlineBetweenTypeDefinitionAndMembers = false }
-
 [<Test>]
 let ``type application including nested multiline function type`` () =
-    forcedLongDefnConfig
+    { config with
```

Comment:
Nitpick but this doesn't comply with the [pull request ground rules](https://fsprojects.github.io/fantomas/docs/contributors/Pull%20request%20ground%20rules.html).

I don't really see a compelling reason to deviate from the standard unit template.
All our tests are the same, I'm not a fan of mixing that up in this PR.